ide/image/src/org/netbeans/modules/image/ImageDataObject.java

```diff
@@ -330,13 +315,8 @@ public ImageWidthProperty() {
 
           /** Gets value of property. Overrides superclass method. */
           public Integer getValue() throws InvocationTargetException {
```

Comment:
Is it intentional, that the `InvocationTargetException` was not removed? This is in contrast to new line 238.

Reply:
Yes, it was intentional.. This would involve a public method change. And while I would like to do it, I didn't think it would get approved with that change.

I'm happy to do it and learn how to make changes like this, but I need a little bit of guidance how to do this.